api-samples/printing/printers.css

```diff
@@ -16,3 +16,7 @@ table {
 div {
   overflow: auto;
 }
+
+#statusDiv {
+  display: none !important;
```

Comment:
why `!important`? its best to avoid using it if possible

Reply:
That's from an earlier iteration of the sample. removing.